src/community/security/oauth2-openid-connect/oauth2-openid-connect-core/src/test/java/org/geoserver/security/oauth2/OpenIdConnectIntegrationTest.java

```diff
@@ -78,7 +78,7 @@ public static void beforeClass() throws Exception {
                                         .withStatus(200)
                                         .withHeader(
                                                 "Content-Type", MediaType.APPLICATION_JSON_VALUE)
-                                        .withBodyFile("jkws.json")));
+                                        .withBodyFile("jwks.json")));
```

Comment:
Corrects the name here but the test resource modified is still named jkws.json? Is it used?

Reply:
The resource has been renamed as well and added a test case.